merkle-patricia-tree/src/Data/Tree/MerklePatricia.hs

```diff
@@ -530,3 +533,33 @@ stripCommonPrefix = \case
 dropPrefix :: String -> String -> String
 dropPrefix pre str =
   fromMaybe str (stripPrefix pre str)
+
+toBits :: ByteString -> String
```

Comment:
Could: if we would stick with patricia tries, the keys could rather be `[Bool]` (or really a `newtype` around that) instead of the much larger `[Char]`.

Reply:
Though, sticking to strings gives us a lot of nice primitives to work with, especially in the land of prefixes and whatnot.

Reply:
Most of them are functions on `[a]` or what would be missing in particular?